operations/deployment/terraform/security-group.tf

```diff
@@ -1,45 +1,50 @@
 resource "aws_security_group" "ec2_security_group" {
-  name        = var.security_group_name
+  name        = "${var.aws_resource_identifier}-instance-sg"
```

Comment:
do SG names not have a size limit?

Reply:
`A name can be up to 255 characters`

I doubt we need to stress the length of this resource